### PR TITLE
Fix e2e tests

### PR DESCRIPTION
**Pull request by darioAnongba**
PR to fix e2e tests by generating new certificates and using the required ones

### CODE REVIEW

File: deps/config/apikey/sample.csv [hunk 1]
```diff
@@ -4,4 +4,4 @@
 # sha256("admin-user")
 f470213d9ae659187a19b9cb2169b4b400544f4d3f59250eda657154700da616;tenant1:user1;*:*;admin
 # sha256("guest-user")
-8605f70ff5f55e2a9323d97de3dbf8e61f38314d93298ca00e19f8918fe8971b;tenant2:user2;*:secrets,*:keys,read:eth1accounts;signer
+8605f70ff5f55e2a9323d97de3dbf8e61f38314d93298ca00e19f8918fe8971b;tenant2:user2;*:secrets,*:keys,read:eth1accounts;
```
Comment (6l20) [inline deps/config/apikey/sample.csv:7]: Last column should be the role here, was it removed in code ?

File: deps/ca/ca.srl [hunk 2]
```diff
@@ -1 +1 @@
-554E0AF33CB76CA1A5B493A14C2F1170C290B4B2
+554E0AF33CB76CA1A5B493A14C2F1170C290B4B4
```
Comment (6l20) [inline deps/ca/ca.srl:1]: Do we really need that srl file here ?

File: deps/config/apikey/sample.csv (hunk 1 above)
Comment (darioAnongba) [reply deps/config/apikey/sample.csv:7]: no role is needed for "guest-user" , this breaks the e2e tests.

File: deps/ca/ca.srl (hunk 2 above)
Comment (darioAnongba) [reply deps/ca/ca.srl:1]: removed

**Review by 6l20 — APPROVED** [review]

**Pull request merged**